annyang.js

```diff
@@ -128,6 +128,15 @@
     invokeCallbacks(callbacks.resultNoMatch, results);
   };
 
+  var logMessage = function(logMsg, emphStyle) {
+    emphStyle = emphStyle || debugStyle;
+    if (/%c/.test(logMsg)) {
```

Comment:
Honestly: This is just crap, and what makes it obvious is that you have to change line 118 to make it actually work. Things that happen during nighttime when motivation and mental performance are flagrantly drifting apart …